src/CommonITILTask.php

```diff
@@ -1662,7 +1662,7 @@ public static function genericDisplayPlanningItem($itemtype, array $val, $who, $
 
         // $val['content'] has already been sanitized and decoded by self::populatePlanning()
         $content = $val['content'];
-        $html .= "<div class='event-description rich_text_container'>" . htmlescape($content) . "</div>";
+        $html .= "<div class='event-description rich_text_container'>" . $content . "</div>";
```

Comment:
I'm not sure escaping should be removed

Reply:
As noted in the PR description, we expect to be showing HTML content. All content is passed through a method to get the safe HTML in the methods that populate the planning data.